packages/records/lib/utils/encryption.ts

```diff
@@ -2,12 +2,20 @@ import type { EncryptedRecordData, FormattedRecord, UnencryptedRecord, Unencrypt
 import { Encryption } from '@nangohq/utils';
 import { envs } from '../env.js';
 
+function getEncryption(): Encryption {
+    const encryptionKey = envs.NANGO_ENCRYPTION_KEY;
+    if (!encryptionKey) {
+        throw new Error('NANGO_ENCRYPTION_KEY is not set');
+    }
+    return new Encryption(encryptionKey);
```

Comment:
Could be done in the construct I guess

Reply:
The thing is the constructor is also used by `shared/lib/utils/encryption.manager.ts` and doing so will change its behavior. So for now it is specific to records